plugins/woocommerce-admin/client/products/fields/attribute-field/attribute-field.tsx

```diff
@@ -231,13 +273,27 @@ export const AttributeField: React.FC< AttributeFieldProps > = ( {
 				) ) }
 			</Sortable>
 			<AddAttributeListItem
+				label={
+					isOnlyForVariations
+						? __( 'Add option', 'woocommerce' )
+						: undefined
+				}
```

Comment:
Here is an example of how we can use the `attributeType` to determine what UI strings to use.